src/js/actions/LayerActions.js

```diff
@@ -30,7 +30,11 @@ class LayerActions {
     return info;
   }
 
-  changeFiresTimeline (selectedIndex) {
+  changeViirsFiresTimeline (selectedIndex) {
```

Comment:
TODO: Decide whether we still need these timelines now that we have the start and end date pickers for each Fires layer. @csphang